js/libs/interface.js

```diff
@@ -327,6 +335,15 @@ const app = new Vue({
             Fliplet.Studio.emit('reload-widget-instance', widgetId)
           }
         })
+    },
+    changeWrapperMarginByErrorsHolderHeight() {
+      this.$nextTick(() => {
+        const wrapper = this.$refs['add-markers-wrapper'];
+        const holder = this.$refs['errors-holder'];
+        if (wrapper && holder) {
+          wrapper.style.paddingBottom = (holder.clientHeight + 'px');
```

Comment:
1. no need for parenthesis here
2. use templated strings for the value

Reply:
@squallstar fixed